base/applications/rapps/unattended.cpp

```diff
@@ -125,7 +122,7 @@ BOOL CALLBACK CmdFindAppEnum(CAvailableApplicationInfo *Info, BOOL bInitialCheck
         return TRUE;
     }
 
-    ConPrintf(StdOut, (LPWSTR)L"%s (%s)\n", (LPCWSTR)(Info->m_szName), (LPCWSTR)(Info->m_szPkgName));
+    ConPrintf(StdOut, L"%s (%s)\n", (LPCWSTR)(Info->m_szName), (LPCWSTR)(Info->m_szPkgName));
```

Comment:
Are the casts for `Info->m_szName` and co. still necessary? (I don't think so)

Reply:
yes they are, since those are CString's

Reply:
I'll change that to `.GetString()` to be a bit more obvious